src/lib/i18n/dictionaries/englishDictionary.ts

```diff
@@ -2106,6 +2106,27 @@ const factions: Record<Factions, string> = {
   "faction.greeting.sunflorians": "Our lineage will reign supreme!",
   "faction.greeting.bumpkins": "We must unite the farmers!",
   "faction.greeting.nightshades": "The shadows will guide us to victory!",
+
+  // Faction Claim Emblems
+  "faction.claimEmblems.alreadyClaimed":
+    "Hey! We already rewarded you with your emblems!",
+  "faction.claimEmblems.emblemsEarned": "Emblems Earned",
+  "faction.claimEmblems.yourRank": "Your Rank",
+  "faction.claimEmblems.yourPercentile": "Your Percentile",
+  "faction.claimEmblems.yourEmblems": "Your Emblems",
+  "faction.claimEmblems.noContribution":
+    "You have not contributed any points to the faction.",
+  "faction.claimEmblems.statistics": "Statistics",
+  "faction.claimEmblems.thankYou":
+    "Thank you {{player}}for your contributions to the {{Faction}}! Take these emblems, they represent your ownership of the faction!",
```

Comment:
Minor typo
```suggestion
    "Thank you {{player}} for your contributions to the {{Faction}}! Take these emblems, they represent your ownership of the faction!",
```

Reply:
@eliasSFL I did this because Without `player` we only want one space in the output.